bigtable/bttest/setup_test.go

```diff
@@ -53,21 +51,6 @@ func newClient(t *testing.T) (context.Context, *clientIntf, bool) {
 	return ctx, cl, false
 }
 
-func dropRowRange(t *testing.T, ctx context.Context, ret *clientIntf) (context.Context, *clientIntf, bool) {
```

Comment:
static file also got this issue, method is not being used anywhere

Reply:
nice